packages/client/src/views/SysAdmin/Config/Application/Tabs/GeneralProperties/GovtLogo.tsx

```diff
@@ -213,7 +212,7 @@ export function GovtLogo() {
             ? intl.formatMessage(messages.applicationConfigUpdatingMessage)
             : notificationStatus === NOTIFICATION_STATUS.SUCCESS
             ? intl.formatMessage(messages.govtLogoChangeNotification)
-            : errorMessages}
+            : intl.formatMessage(messages.govtLogoChangeError)}
```

Comment:
@modupeadeonojobi There are more than one error that could happen when uploading the govt logo for example the uploaded file size could be more than what's allowed and the `errorMessages` state is set to the correct message accordingly. So instead of setting the error message directly here, could you look for the place where the upload error occurred and set error message appropriately using `setErrorMessages`?

Reply:
Okay, will do